app/models/outcome_coverage.rb

```diff
@@ -1,7 +1,10 @@
 class OutcomeCoverage < ActiveRecord::Base
   belongs_to :course
-  belongs_to :outcome
-  belongs_to :subject
+  belongs_to :outcome, required: true
+  belongs_to :subject, required: true
```

Comment:
Yeah, it's the default for Rails 5 generated apps, but when you update from Rails 4 it adds this: https://github.com/MIT-IR/abet/blob/master/config/initializers/new_framework_defaults.rb

That maintains the Rails 4 behavior. I tried removing that when we upgraded to Rails 5 but I got a bunch of test failures I didn't want to look into. That could be something we do when we're looking for an activity to round out a day.